statsmodels/genmod/generalized_estimating_equations.py

```diff
@@ -1911,11 +1908,21 @@ def qic(self, scale=None):
         """
         Returns the QIC and QICu information criteria.
 
-        For families with a scale parameter (e.g. Gaussian),
-        provide as the scale argument the estimated scale
-        from the largest model under consideration.
+        For families with a scale parameter (e.g. Gaussian), provide
+        as the scale argument the estimated scale from the largest
+        model under consideration.
+
+        If the scale parameter is not provided, the estimated scale
+        parameter is used.  Doing this does not allow comparisons of
+        QIC values between models.
         """
 
+        # It is easy to forget to set the scale parameter.  Sometimes
+        # this is intentional, so we warn.
+        if isinstance(self.family, families.Gaussian) and scale is None:
```

Comment:
GEE also allows other families with scale != 1

https://www.statsmodels.org/dev/_modules/statsmodels/genmod/generalized_estimating_equations.html#GEE.estimate_scale
lists the families with scale=1, instead of the complement set

in general there should be a scale option as in GLM, e.g. to handle purely overdispersed Poisson (i.i.d. with scale!=1).